### PR TITLE
BUG: Fix issues with Markups plane interaction scale handles

### PR DESCRIPTION
**Pull request by Sunderlandkyl**
This commit solves 2 issues:
- When loading scenes with a plane of area 0.0mm, the scale handle points were not fully initialized, which could cause a crash. Fixed by always initializing the points, and instead hiding the scale handles during UpdateHandleVisibility for planes with area 0.0mm.
- Scale handles were not displayed at the incorrect locations in 3D while the plane was being moved. Fixed by calling Superclass::UpdateInteractionPipeline() before updating the scale handle, ensuring that the HandleToWorldTransform was updated before calculating scale handle position.

### CODE REVIEW

File: Modules/Loadable/Markups/VTKWidgets/vtkSlicerPlaneRepresentation3D.cxx [hunk 1]
```diff
@@ -330,6 +330,9 @@ void vtkSlicerPlaneRepresentation3D::UpdateFromMRML(vtkMRMLNode* caller, unsigne
 //----------------------------------------------------------------------
 void vtkSlicerPlaneRepresentation3D::UpdateInteractionPipeline()
 {
+  // Final visibility handled by superclass in vtkSlicerMarkupsWidgetRepresentation
```
Comment (lassoan) [inline Modules/Loadable/Markups/VTKWidgets/vtkSlicerPlaneRepresentation3D.cxx:333]: Add a note in the comment that explains why this call must be made first.

**Review by lassoan — COMMENTED** [review]
Thank you, these look good. I've just added a small suggestion to add in a comment why we need to put that line first (to ensure that the code will not be moved to incorrect location during future refactorings).

File: Modules/Loadable/Markups/VTKWidgets/vtkSlicerPlaneRepresentation3D.cxx (hunk 1 above)
Comment (Sunderlandkyl) [reply Modules/Loadable/Markups/VTKWidgets/vtkSlicerPlaneRepresentation3D.cxx:333]: Thanks! I've added a comment.

**Review by lassoan — APPROVED** [review]
Looks perfect, thank you.

**Pull request merged**